From 2ebadefad7d5b8d6770fc135f1ad48059a5ac3b5 Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Fri, 17 Aug 2018 20:12:53 -0700
Subject: [PATCH] ABI-Testsuite: Force the old c++11 ABI in mangling/test.xpp

The check lines for this test assume the old c++11 ABI.
---
 ABI-Testsuite/test/mangling/test.xpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ABI-Testsuite/test/mangling/test.xpp b/ABI-Testsuite/test/mangling/test.xpp
index e7f4e00..b39e284 100755
--- a/ABI-Testsuite/test/mangling/test.xpp
+++ b/ABI-Testsuite/test/mangling/test.xpp
@@ -1,6 +1,6 @@
 // This file is distributed under the University of Illinois Open Source License.
 // See LICENSE.TXT for details.
-// RUN: cxx_compiler cxx_11 -c %s -o %t.o
+// RUN: cxx_compiler cxx_11 -D_GLIBCXX_USE_CXX11_ABI=0 -c %s -o %t.o
 // RUN: bindump %t.o | FileCheck -allow-deprecated-dag-overlap prefixes %s
 
 #include <cstddef>
-- 
1.8.3.1

